Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:20-alpine
+
+WORKDIR /app
+COPY package.json .
+
+RUN yarn install
+
+# Bare neccesary
+COPY src/ ./src
+COPY .env .
```

Comment:
```suggestion
```

Some values in the `.env` file might be secrets, so you should mount them at runtime instead of put them in the built and published image.

Using [`env_file`](https://docs.docker.com/reference/compose-file/services/#env_file) in the compose file instead.